assets/src/edit-story/components/library/panes/pageTemplates/templateList.js

```diff
@@ -84,14 +84,16 @@ function TemplateList({ pages, parentRef, pageSize }) {
   });
 
   const handleKeyboardPageClick = useCallback(
-    ({ key }, page) => {
-      if (key === 'Enter') {
-        if (isGridFocused) {
+    ({ key, code }, page) => {
+      if (isGridFocused) {
+        if (key === 'Enter') {
           handlePageClick(page);
+        } else if (code === 'Space') {
```

Comment:
Is that how we do this? Is that intuitive for keyboard users? Enter activates the whole item and space activates the menu? I'm okay with it, but just a bit surprised about this pattern.

Reply:
That's how media works, too.